odh/overlays/moc/zero/kafka/overrides/kafka/overlays/topics/zero-cluster-app-logs.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: zero-prod.cluster.app-logs
+  labels:
+    strimzi.io/cluster: odh-message-bus
+spec:
+  partitions: 1
```

Comment:
This doesn't seem very resilient.

Reply:
How many partitions would you recommend?

Reply:
I'd also go with at least 2 replicas

Reply:
updated :+1:

Reply:
I don't think this effect resilience as much as it effects performance. You need to keep in mind how many message producers and how many consumers you want to have for that topic. Also mind that messages on different partition are not ordered. So in case you want to ensure message ordering and you expect to have a single consumer, it doesn't make sense for it to listen on 3 different partitions.

What are you expectations @4n4nd ? How many pods you expect to produce messages here and how many pods will be consuming the messages?